app/src/main/res/layout/resume_lesson_activity.xml

```diff
@@ -21,7 +21,7 @@
         android:background="@color/component_color_shared_activity_toolbar_color"
         android:minHeight="?attr/actionBarSize"
         android:theme="@style/Widget.AppCompat.ActionBar"
-        app:navigationContentDescription="@string/navigate_up"
+        app:navigationContentDescription="@string/hints_andSolution_close_icon_description"
```

Comment:
I think this is out of scope for this PR. Please remove this change.

This is the resume lesson screen:
![Screenshot_1689663974](https://github.com/oppia/oppia-android/assets/59600948/30306a2a-4e40-42e5-9856-8af99a6a3d78)

Reply:
This is also resulting in test failure:

```
1) testResumeLessonActivity_onToolbarClosePressed_closesResumeExplorationActivity(org.oppia.android.app.resumelesson.ResumeLessonActivityTest)
androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: with content description from resource id: <2131821006>[nav_app_bar_navigate_up_description] value: Navigate up
```

Reply:
I dont know why but in my app the back button is replaced with the close icon as it is in the hint.
Let me just confirm.

Reply:
Done.